tests/modules/programs/inori/full-config.nix

```
@@ -0,0 +1,46 @@
+{ config, ... }:
+
+{
+  config = {
+    programs.inori = {
+      enable = true;
+      patch_array_keybindings = true;
+      settings = {
+        seek_seconds = 10;
+        dvorak_keybindings = true;
+        keybindings = {
+          toggle_playpause = [ "p" "<space>" ];
+          next_song = [ ">" "C-n" ];
+          previous_song = [ "<" "C-p" ];
+          seek = "<right>";
+          seek_backwards = "<left>";
+
+          up = "k";
+          down = "j";
+          left = "h";
+          right = "l";
+          top = [ "g g" "<home>" ];
+          bottom = [ "G" "<end>" ];
+          quit = [ "<esc>" "q" ];
```

```suggestion
          quit = [ "<escape>" "q" ];
```

From https://github.com/eshrh/inori/commit/0e371e38c54f6e170e966da0ee1bfe357d59f122, as `<esc>` is inconsistent with the rest of the special keys (which are full words).

---

the keybind arrays (https://github.com/eshrh/inori/commit/b6f7b5d3333312231ad8f13e228f190920d19938) are also blocked on a new release, right?

should be pretty fast as @eshrh just needs to tag a new release on github/cargo and on the nixpkgs side we can merge without approval using the merge bot.

---

yeah I'll put these config updates in a new release. can't say much when my bff wants one :3

---

> the keybind arrays (https://github.com/eshrh/inori/commit/b6f7b5d3333312231ad8f13e228f190920d19938) are also blocked on a new release, right?

right completely forgot about that